data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/special_damage_type.cfg

```diff
@@ -196,7 +196,7 @@
 # Move Alice next to Bob, and have Alice attack Bob.
 ##
 # Expected end state:
-# Alice attack with fire; and Bob use cold because Bob [damage] filter must check type=blade and not the replacement type. That's partly because it could otherwise cause infinite recursion, and partly because having a set limit to the number of recursions would make it unpredicatable which specials are active.
+# Alice attack with fire and Bob use cold because Bob [damage] filter must check type=blade and not the replacement type. That's partly because it could otherwise cause infinite recursion, and partly because having a set limit to the number of recursions would make it unpredicatable which specials are active.
```

Comment:
```suggestion
# Alice attacks with fire and Bob uses cold because Bob's [damage] filter must check type=blade and not the replacement type. That's partly because it could otherwise cause infinite recursion, and partly because having a set limit to the number of recursions would make it unpredictable which specials are active.
```